core/templates/components/common-layout-directives/common-elements/sharing-links.component.ts

```diff
@@ -15,18 +15,20 @@
 /**
  * @fileoverview Component for the Social Sharing Links.
  */
-const constants = require('constants.ts');
+
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
 
 import { Component, Input, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 
-import { UrlInterpolationService } from
-  'domain/utilities/url-interpolation.service';
-import { SiteAnalyticsService } from 'services/site-analytics.service';
-import { HtmlEscaperService } from 'services/html-escaper.service';
 import { ExplorationEmbedButtonModalComponent } from
   'components/button-directives/exploration-embed-button-modal.component';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
```

Comment:
Why are these here and `UrlInterpolationService` is at the top?

Reply:
@vojtechjelinek All of these are grouped in this order ["builtin", "external", "parent", "sibling", "index"]. You can see the configuration here (https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md)